docs/sections/user_guide/cli/drivers/mpas.rst

```diff
@@ -0,0 +1,109 @@
+``mpas``
+==========
+
+The ``uw`` mode for configuring and running the MPAS forecast model. Any of the ``TASK`` arguments listed below may be called to generate that component required for the ultimate ``run`` task. A ``provisioned_run_directory`` will perform all of the tasks except for running the executable.
```

Comment:
```suggestion
The ``uw`` mode for configuring and running the MPAS forecast model. Each listed ``TASK`` may be called to generate the runtime asset(s) it is responsible for, and will call any task it depends on as needed. A ``provisioned_run_directory`` comprises everything needed for a run, and a ``run`` runs the MPAS executable.
```
I don't know how much better this is. Use any/all/nothing as you like.

Reply:
This makes a lot more sense! Thanks.